demisto_sdk/commands/lint/resources/pylint_plugins/base_checker.py

```diff
@@ -449,6 +449,47 @@ def _check_if(comp_with):
         except Exception:
             pass
 
+    # -------------------------------------------- Match Node ---------------------------------------------
+
+    def _commands_in_match_statement_checker(self, node):
```

Comment:
please add typing, also, will it work only on python 3.10? what happens if the python version < 3.10?
this feature is relevant only for python 3.10 if am correct